doc/src/manual/functions.md

```diff
@@ -1035,7 +1035,7 @@ julia> ["a", "list", "of", "strings"] .|> [uppercase, reverse, titlecase, length
  7
 ```
 
-When combining pipes with anonymous functions, parentheses must be used if subsequent pipes are not to be parsed as part of the anonymous function's body. Compare:
+When combining pipes with anonymous functions, it is mandatory to use parentheses if the subsequent pipes are not to parse as part of the private function's body. Compare:
```

Comment:
The original sentence is fine as-is. #47875 fixed the typo mentioned in #51442 an it will show up correct once 1.10 is released.

Reply:
I revert the changes thank you